ui/src/buckets/selectors/index.ts

```diff
@@ -0,0 +1,32 @@
+// Types
+import {Bucket} from 'src/types'
+
+export const SYSTEM = 'system'
+
+export const isSystemBucket = (type: string): boolean => type === SYSTEM
+
+export const sortBucketNames = (buckets: Bucket[]) =>
```

Comment:
this is sorting buckets by name alphabetically. I think it might be better to call it `sortBucketsByName`